### PR TITLE
feat(form): enable hint messages on questions

### PR DESCRIPTION
**Pull request by luytena**
Add a hint text field on all question types except for
form, static and action button questions. This functionality
requires a db migration.

### CODE REVIEW

**Review by czosel — APPROVED** [review]
LGTM :+1: @winged or @open-dynaMIX, could you take a quick look as well?

File: caluma/caluma_form/serializers.py [hunk 1]
```diff
@@ -243,7 +245,7 @@ def validate(self, data):
         return super().validate(data)
 
     class Meta(SaveQuestionSerializer.Meta):
-        fields = SaveQuestionSerializer.Meta.fields
+        fields = SaveQuestionSerializer.Meta.fields + ["hint_text"]
```
Comment (winged) [inline caluma/caluma_form/serializers.py:248]: Should probably be moved to `SaveQuestionSerializer.Meta`, so you won't have to explicitly extend the fields list in every subtype

**Review by winged — CHANGES_REQUESTED** [review]
Code generally looks good except a small cleanup. However I'm wondering what is the use of this? What's the use case of hint vs. info texts?

File: caluma/caluma_form/serializers.py (hunk 1 above)
Comment (winged) [reply caluma/caluma_form/serializers.py:248]: Ah, nevermind, needs to be this way because form, static and action buttons don't have hints

**Review by winged — APPROVED** [review]

**Comment by winged** [comment]
Ah, ok, makes sense!

**Pull request merged**